repositories/droplet_repository_test.go

```diff
@@ -0,0 +1,244 @@
+package repositories_test
+
+import (
+	"context"
+	"time"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+
+	workloadsv1alpha1 "code.cloudfoundry.org/cf-k8s-controllers/apis/workloads/v1alpha1"
+
+	. "code.cloudfoundry.org/cf-k8s-api/repositories"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ = Describe("DropletRepository", func() {
+	Describe("GetDroplet", func() {
```

Comment:
The method is called `FetchDroplet`, which doesn't match. We've used Fetch elsewhere, so we should probably change this line and not rename the method.

Reply:
Addressed on commit [8a5dfced](https://github.com/cloudfoundry/cf-k8s-api/pull/131/commits/8a5dfced2f8a16a0d25310f384e2cb265d2a4a7d)